src/Components/ButtonPanel.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class ButtonPanel extends Component {
+  render() {
+    return <div className="Button-Panel">{ this.props.children }</div>;
+  }
+}
+
+ButtonPanel.propTypes = { children: PropTypes.string };
```

Comment:
- Let's make this a functional component
- You need to be passing a prop to your button compoenent.
- It needs to be rendering the Button compoenent as part of the ButtonPanel.
![image](https://user-images.githubusercontent.com/46115725/89369077-b94b7e80-d6a2-11ea-87ba-a1b57988fd42.png)